components/automate-ui/src/app/modules/infra-proxy/chef-servers-list/chef-servers-list.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ChefServersListComponent } from './chef-servers-list.component';
+
+xdescribe('ChefServersListComponent', () => {
+  let component: ChefServersListComponent;
+  let fixture: ComponentFixture<ChefServersListComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ ChefServersListComponent ],
+      schemas: [ CUSTOM_ELEMENTS_SCHEMA ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ChefServersListComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Please add some tests. The functionality of the underlying component is simple, yes, but should still be tested. You should be able to adapt existing tests for other components that do very similar things.

Reply:
Nice tests; thank you!